### DISCUSSION TITLE
did the logo change?

### DISCUSSION THREAD

**Discussion by dalers** [timschofield/webERP#318] [Development Discussion & Specification]
Just noticed this....

- Why did the log change?
- What does this logo represent?

![image](https://github.com/user-attachments/assets/87583d07-e6f5-40db-956f-45b3cda542bc)

**Comment by dalers** [comment]
Fwiw, I prefer the previous logo, but wouldn't object if the new logo had a rounded rectangle instead of a "test tube" look (being chariatable). 

Alternatively, the logo Desarrolladores created has a traceable origin and free license https://commons.wikimedia.org/wiki/File:WebERP_logo.svg

**Comment by timschofield** [comment]
It was this commit -
https://github.com/timschofield/webERP/commit/5e6ce4dff525b17905ccadd9aeaadbc1b762406c
- where it goes wrong. web and ERP need to be in separate <div> tags.

I have fixed this

On Wed, 18 Dec 2024 at 21:18, Dale Scott ***@***.***> wrote:
>
> Just noticed this....
>
> Why did the log change?
> What does this logo represent?
>
> image.png (view on web)
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>



-- 
www.weberp.org
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Comment by dalers** [comment]
Wow, I couldn't have been more wrong as to what actually happened! Confirmed fixed. Thanks Tim.
